### DISCUSSION TITLE
Reduce ChatGPT 3.5 token consumption

### DISCUSSION THREAD

**Discussion by hjw21century** [lobehub/lobehub#1896] [General Question]
Background: 
Currently, ChatGPT 3.5 is open to unregistered users, so using the official unregistered approach to use ChatGPT 3.5 can reduce unnecessary token consumption. 

Refer:
The ChatGPT 3.5 API service can be set up by yourself, refer to https://github.com/aurora-develop/aurora.

**Comment by arvinxx** [comment]
You can set it as proxy. But I won't add it natively in LobeChat. 

ChatGPT reverse proxy don't supported function call. So the plugins are disabled.
